test/NameBinding/named_lazy_member_loading_objc_protocol.swift

```diff
@@ -0,0 +1,19 @@
+// REQUIRES: objc_interop
+// REQUIRES: OS=macosx
+// RUN: rm -rf %t && mkdir -p %t/stats-pre && mkdir -p %t/stats-post
+//
+// Prime module cache
+// RUN: %target-swift-frontend -typecheck %s
+//
+// Check that without lazy named member loading, we're importing too much.
+// RUN: %target-swift-frontend -typecheck -stats-output-dir %t/stats-pre %s
+// RUN: %utils/process-stats-dir.py --evaluate 'NumTotalClangImportedEntities > 50' %t/stats-pre
+//
+// Check that with lazy named member loading, we're importing less.
+// RUN: %target-swift-frontend -typecheck -enable-named-lazy-member-loading -stats-output-dir %t/stats-post %s
+// RUN: %utils/process-stats-dir.py --evaluate 'NumTotalClangImportedEntities < 10' %t/stats-post
```

Comment:
It seems brittle to depend on WebKit not to change. Can you make a fake test case instead?

Reply:
I .. guess? The synthetic testcase is the other one above (it includes a stats measurement). I was hoping to "check that it matters in the real world too" but I guess I can just omit this test.